README.md

```diff
@@ -1,3 +1,5 @@
+# THIS PACKAGE IS NO LONGER MAINTAINED
```

Comment:
Perhaps we do a little better than this? I don't believe that this addon ever worked beyond a single rule so we should do everything we can to discourage usage.

Reply:
Sure, we can chat in IRL in a few mins 😄

Reply:
@GavinJoyce I've updated the README text